jcl/src/com.ibm.management/share/classes/com/ibm/lang/management/internal/DiagnosticsMXBeanImpl.java

```diff
@@ -0,0 +1,150 @@
+/*[INCLUDE-IF Sidecar17]*/
+/*******************************************************************************
+ * Copyright (c) 2017, 2017 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+
+package com.ibm.lang.management.internal;
+
+import javax.management.MalformedObjectNameException;
+import javax.management.ObjectName;
+
+import com.ibm.jvm.Dump;
+import com.ibm.jvm.DumpConfigurationUnavailableException;
+import com.ibm.jvm.InvalidDumpOptionException;
+
+import com.ibm.jvm.Trace;
+import com.ibm.lang.management.DiagnosticsMXBean;
+
+public class DiagnosticsMXBeanImpl implements DiagnosticsMXBean {
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void resetDumpOption() throws DumpConfigurationUnavailableException {
+		Dump.resetDumpOptions();
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void configureDumpOptions(String dumpOptions) throws InvalidDumpOptionException, DumpConfigurationUnavailableException {
```

Comment:
Should this be called setDumpOptions instead of configureDumpOptions? According to the MXBean naming conventions, a setter should be called set...
As a practical example, jconsole handles the APIs differently when they are getters, setters and boolean APIs,